integration/time_drift/time_drift.bats

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will make sure that guest time
+# is synced with the host
+
+set -e
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "Verify guest time is synced with the host" {
+	CONTAINER_NAME="test"
+	IMAGE="busybox"
+	PAYLOAD="tail -f /dev/null"
+
+	docker run -d --name "${CONTAINER_NAME}" --runtime "${RUNTIME}" "${IMAGE}" sh -c "${PAYLOAD}"
+
+	# Get host and guest time
+	GUEST_TIME=$(docker exec "${CONTAINER_NAME}" date)
```

Comment:
I'm still not sure this is doing what you think maybe? Have you tried this under a `bash -x` to see what it looks like? I could be wrong, but it just doesn't look right to my eyes...

I suspect it is currently:
 - executing the docker exec, and placing that date into GUEST_TIME
 - executing `date` and placing that into HOST_TIME
 - attempting to execute the contents of GUEST_TIME and HOST_TIME, and piping the output of that execution to `parallel`
 - and finally doing the date compare.

What I suspect you mean it to do is:
 - set GUEST_TIME to a string of commands to execute
 - set HOST_TIME to a string of commands to execute
 - execute them at the same time using `parallel` (I'm not sure you can use `|` in the style you are trying, unless you `echo` stuff ...?)
 - and then do the compare of the results (for which you might need to save them in two new variables??)

Probably best to show us what a `bash -x <fragment>` looks like so we can see the order of events and the values/variables set and used.